sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -465,12 +465,12 @@ To get all properties, use rest syntax:
 let { a, b, c, ...everythingElse } = $props();
 ```
 
-If you're using TypeScript, you can use type arguments:
+If you're using TypeScript, you can declare the prop types:
 
 ```ts
 type MyProps = any;
 // ---cut---
-let { a, b, c, ...everythingElse } = $props<MyProps>();
+let { a, b, c, ...everythingElse }: MyProps = $props();
 ```
 
```

Comment:
```suggestion

> In an earlier version of the Svelte 5 preview, `$props()` took a generic argument and you could type it as `let { .. } = $props<MyProps>();`. This is no longer possible since it's prone to introduce bugs, because default initializers can widen the type.
```

Reply:
good shout. i'm going to add an actual code example, because 'default initializers can widen the type' is the sort of thing i would stare at for a while before understanding what it meant